packages/theme/modules/catalog/pricing/getGroupedProductPriceCommand.ts

```diff
@@ -8,8 +8,8 @@ export function getGroupedProductPriceCommand(product: GroupedProduct): number {
     return regular > special && special !== null ? special : regular;
   };
 
-  return product.items.reduce(
+  return product.items ? product.items.reduce(
```

Comment:
```suggestion
  return product.items?.reduce(...) ?? 0
```


sry for 2nd round, didn't notice before

Reply:
Fixed.